internal/driver/commands.go

```diff
@@ -199,6 +199,14 @@ var configHelp = map[string]string{
 		"Use name=value syntax to limit the matching to a specific tag.",
 		"Numeric tag filter examples: 1kb, 1kb:10kb, memory=32mb:",
 		"String tag filter examples: foo, foo.*bar, mytag=foo.*bar"),
+	"tagleaf": helpText(
+		"Adds a pseudo leaf node for samples with tags matched by regexp",
+		"Use name=value1,value2 syntax to specify the tag and the values",
+		"for which nodes would be added.  Example: latency=1s,2s,3s."),
+	"tagroot": helpText(
+		"Adds a pseudo root node for samples with tags matched by regexp",
+		"Use name=value1,value2 syntax to specify the tag and the values",
+		"for which nodes would be added.  Example: latency=1s,2s,3s."),
```

Comment:
Couple of questions:

1) The option / flag is of type []string / multi-value. The examples are of the form "foo=val1,val2" What exactly the multi-value here? Is it the comma-separated multi-values as in example, i.e. "foo=val1" and "val2" as two values which is weird? Or is that there is ability to have multiple occurrences of each of these like "-flag=foo=val1,val2 -flag=bar=val3,val4" and the documentation needs to explain this?

2) How are numeric tags handled, including with units like memory? Can we reuse the same value format as in tagfocus / taghide etc.? I feel that we are introducing a new way to specify a set of values and I don't see why different tag* flags can't unify on a single way to do that.